### ISSUE TITLE
block code - kotlin Syntax highlighting and nested comment little problem

### ISSUE THREAD

**Issue by bertuzziclaudio** [laurent22/joplin#1788]
## Environment

Joplin version: 1.0.161
Platform: Only tested on: Linux,Windows,Android
OS specifcs: Windows 10,Android 8.1 , Kubuntu 19.04

```kotlin
/* LEVEL 0
    /*
      multiline nested comment LEVEL 1
    */
    println("on kotlin this is a commented code in comment LEVEL 0")
    /*
        another multiline nested comment LEVEL 1
    */
*/
```
Describe what you expected to happen:

I espect the block is colorize same as single comment  but the line that start with println is colorized as a executable code (sorry for my bad english)

!!! Also in this post the line is wrong colorized as executable

**Comment by tessus** [comment]
We are using [hightlightjs](https://github.com/highlightjs/highlight.js) and this is an upstream issue. Can you please report the issue there?

See https://jsfiddle.net/tessus/u4o1s5ez/

**Comment by bertuzziclaudio** [comment]
ok 
https://github.com/highlightjs/highlight.js/issues/2092

**Comment by tessus** [comment]
Thanks, let's hope they fix it. There have been PRs open for 4-5 years, I don't even want to start with issues.

**Comment by tessus** [comment]
@laurent22 I'm not sure, but maybe we should switch to a different highlighter. I have the nagging feeling that the highlightjs project is going down the drain.

**Comment by laurent22** [comment]
@tessus, maybe but not sure anything better and highlight.js is widely used so there's a good chance it will keep being maintained.

**Comment by tessus** [comment]
@laurent22 I understand, I just had the feeling that nothing is happening over there. Although they have fixed the nested kotlin comment bug 2 days ago, which seems extremely fast, since there are issues and PRs open that are a few years old.

**Comment by tessus** [comment]
Note to self: the fixd should be available in the release that comes after 9.15.9
